src/pages/index.js

```diff
@@ -1,93 +1,225 @@
 import "../pages/index.css";
 
 import {
-  initialCards,
   selectors,
   config,
   profileEditButton,
   addNewCardButton,
+  changeImageButton,
 } from "../utils/constants.js";
 
+import Api from "../components/Api.js";
+
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
```

Comment:
Please, delete `this._disableButton();` from line 86 in `FormValidator.js` because you shouldn't disable the button immediately after submitting: in case of a server error you should be able to press the button for the 2nd time (it should be active in that case)